server/views/pages/activities/create-an-activity/check-answers.njk

```diff
@@ -190,7 +190,7 @@
                     actions: {
                         items: [
                             {
-                                href: "end-date?preserveHistory=true",
+                                href: "remove-end-date" if session.createJourney.endDate else "end-date?preserveHistory=true",
```

Comment:
Realised you need a feature toggle in case this gets deployed to `pre-prod`/`prod` before testing in `dev`